Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/resource/helper.go

```diff
@@ -117,7 +117,7 @@ func (m *Helper) Create(namespace string, modify bool, data []byte) (runtime.Obj
 			if err := m.Versioner.SetResourceVersion(obj, ""); err != nil {
 				return nil, err
 			}
-			newData, err := m.Codec.Encode(obj)
+			newData, err := m.RESTClient.GetCodec().Encode(obj)
```

Comment:
so leave that alone for now and just refactor the internal methods to let you pass through the []byte or runtime.Object. The helper's codec then is only used for decoding in order to set version when `modify` is true